### PR TITLE
German partner update

### PR DESCRIPTION
**Pull request by jutta-frieden**
Translated the new partner page into German.

### CODE REVIEW

**Comment by jutta-frieden** [comment]
@georg911 Could you proof-read the partner page please? Thank you!
@samjewell Could you check the code is ok please? Cheers

**Comment by samjewell** [comment]
@jutta-frieden would you mind putting a screenshot of the page into the PR comments too? Just makes it a little easier to review - thanks!

File: app/pages/partners/partners.de.js [hunk 1]
```diff
@@ -11,57 +25,377 @@ export default class PartnersDe extends React.Component {
...
+              <h2 className='u-text-heading u-color-heading u-text-light u-text-l u-margin-Bl'>
+                Clubs &amp; Vereine
+              </h2>
+              <p className='u-text-s u-color-p u-margin-Bm'>
+                teamup wurde Partner von GoCardless, um Fitnessstudios und Clubs das Management
+                von Mitgliedern und Abo-Zahlungen zu erleichtern.
+              </p>
+              <p className='u-text-s u-text-light u-text-italic u-margin-Bxs partners-page__quote'>
+                “Unsere Kunden schätzen die Benutzerfreundlichkeit von GoCardless sehr. Als Partner haben sie uns
+                in Großbritannien und Europa entscheidend zu mehr Wachstum verholfen.”
```
Comment (samjewell) [inline app/pages/partners/partners.de.js]: loving the directional speechmarks here ;-)

Comment (jutta-frieden) [inline app/pages/partners/partners.de.js]: partners-germany-4
partners-germany-2
partners-germany-3
partners-germany-1

**Comment by georg911** [comment]
all sweet on the German front @jutta-frieden

**Comment by samjewell** [comment]
Hey - great work on this! There's a couple of things you could improve if you felt like it, or you could leave them if you just want to get this work in.

1. The Zuora logo is very close to the `<hr>` below it. Might be nicer to reduce the padding above "Finanz- und Rechnungswesen" to give the logo a bit more room?
screen shot 2015-09-08 at 10 03 56

2. The splash pages just about work on iPad in landscape mode - but the word Wohltätigkeitsorganisationen is no longer centered in its box because it is so long! Not sure if you could use a shorter word here perhaps, or reduce the padding so it is centered? Just a thought anyway - fine details, and I'm happy for you to merge in as is.
screen shot 2015-09-08 at 10 16 15


:+1: merge when ready!

**Pull request merged**